### PR TITLE
added new nugets

### PR DESCRIPTION
**Pull request by BelehovEgor**
added Image and Broker Supports

### CODE REVIEW

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        userId = _httpContext.GetUserId();
+      }
+
+      if (includeIsAdminCheck && await IsUserAdminAsync(userId.Value))
+      {
+        return true;
+      }
+
+      var result = _requestClientCR.GetResponse<IOperationResult<bool>>(
+          ICheckUserRightsRequest.CreateObj(userId.Value, rightIds),
```
Comment (Cheephot) [inline src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs]: fix tabs

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+using MassTransit;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.Kernel.BrokerSupport.AccessValidatorEngine
+{
+  /// <inheritdoc/>
```
Comment (Cheephot) [inline src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs]: need these comments?

File: src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      {
+        assemblies.Add(Assembly.LoadFrom(fileName));
+      }
+
+      List<Type> parsedEntities = new();
+
+      foreach (var assembly in assemblies)
+      {
+        parsedEntities.AddRange(assembly.ExportedTypes
+            .Where(
```
Comment (Cheephot) [inline src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs]: fix tabs

File: src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+                    && t.GetCustomAttribute(typeof(ParseEntityAttribute)) != null)
+            .ToList());
+      }
+
+      foreach (var entity in parsedEntities)
+      {
+        var attr = entity.GetCustomAttribute<ParseEntityAttribute>();
+
+        var parsedProperties = entity
+            .GetProperties()
```
Comment (Cheephot) [inline src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs]: fix tabs

**Review by Cheephot — APPROVED** [review]

File: src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs (hunk 3 above)
Comment (BelehovEgor) [reply src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs]: fixed

File: src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs (hunk 4 above)
Comment (BelehovEgor) [reply src/Kernel.BrokerSupport/Attributes/ParseEntity/Models/Responses/IFindParseEntitiesResponse.cs]: fixed

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs]: fixed

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs]: fixed

**Review by MariyaSin — APPROVED** [review]

**Review by asyagraf — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    {
+      Response<IOperationResult<bool>> result =
+        await _requestClientUS.GetResponse<IOperationResult<bool>>(
+          ICheckUserIsAdminRequest.CreateObj(userId),
+          timeout: RequestTimeout.After(s: 2));
+
+      if (result.Message == null)
+      {
+        _logger.LogWarning("Failed to send request to UserService via the broker.");
+
```
Comment (VladSpitsin) [inline src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs:31]: remove empty line

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+      }
+
+      var result = await _requestClientCR.GetResponse<IOperationResult<bool>>(
+        ICheckUserRightsRequest.CreateObj(userId.Value, rightIds),
+        timeout: RequestTimeout.After(s: 2));
+
+      if (result.Message == null)
+      {
+        _logger.LogWarning("Failed to send request to CheckRightService via the broker.");
+
```
Comment (VladSpitsin) [inline src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs:89]: remove empty line

File: src/Kernel.BrokerSupport/Attributes/AutoInjectRequestAttribute.cs [hunk 7]
```diff
@@ -0,0 +1,29 @@
+﻿using MassTransit;
+using System;
+
+namespace LT.DigitalOffice.Kernel.BrokerSupport.Attributes
+{
+  [AttributeUsage(AttributeTargets.Property)]
+  public class AutoInjectRequestAttribute : Attribute
+  {
+    public Type Model { get; init; }
+
```
Comment (VladSpitsin) [inline src/Kernel.BrokerSupport/Attributes/AutoInjectRequestAttribute.cs]: remove empty line

File: src/Kernel.BrokerSupport/Broker/IOperationResult.cs [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+namespace LT.DigitalOffice.Kernel.BrokerSupport.Broker
+{
+  /// <summary>
+  /// Interface that is needed to form the response in message broker.
+  /// </summary>
+  public interface IOperationResult<T>
+  {
+    bool IsSuccess { get; }
+
```
Comment (VladSpitsin) [inline src/Kernel.BrokerSupport/Broker/IOperationResult.cs]: remove empty line

File: src/Kernel.BrokerSupport/Broker/IOperationResult.cs [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+    List<string> Errors { get; }
+
```
Comment (VladSpitsin) [inline src/Kernel.BrokerSupport/Broker/IOperationResult.cs]: same

**Review by VladSpitsin — APPROVED** [review]

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs (hunk 5 above)
Comment (MariyaSin) [reply src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs:31]: no need in this plase

File: src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs (hunk 6 above)
Comment (MariyaSin) [reply src/Kernel.BrokerSupport/AccessValidatorEngine/AccessValidator.cs:89]: no need in this plase

**Pull request merged**